bin/annotate

```diff
@@ -22,6 +22,7 @@ has_set_position = {}
 target_action = :do_annotations
 positions = %w(before top after bottom)
 
+# rubocop:disable Metrics/BlockLength
```

Comment:
[note]

finally it reached to 
```
bin/annotate:25:1: C: Metrics/BlockLength: Block has too many lines. [145/142]
```
and I thought it's rational to suppress here IMO